dns/zone-configs/k8s.io.yaml

```diff
@@ -247,6 +247,10 @@ kind.sigs:
 cluster-api.sigs:
   type: CNAME
   value: kubernetes-sigs-cluster-api.netlify.com.
+# https://github.com/kubernetes-sigs/cluster-api v1alpha1 docs (@dwat, @jdetiber, @justinsb)
+v1alpha1.cluster-api.sigs:
+  type: CNAME
+  value: release-0.1--kubernetes-sigs-cluster-api.netlify.com.
```

Comment:
does this actually exist?

Reply:
It does resolve, but we're still working through the rest of the Netlify config, so the content is not yet deployed.

Reply:
Do you foresee that we'd need to this with a number of domains?

Reply:
@cblecker over time, yes. The idea is that we would be able to have docs per minor release and/or API version, not dissimilar to Kubebuilder or Kubernetes proper.

Reply:
Should we do this similar then to the way proper kubernetes docs does it, with a wildcard?

```
# All docs subdomains get sent to netlify (@chenopis)
'*.docs':
  type: CNAME
  value: netlifyglobalcdn.com.
```

Reply:
@davidewatson thoughts?

Reply:
I think this is a good idea since then we wouldn't have to update this repository for each version. Let's decide on https://github.com/kubernetes-sigs/cluster-api/issues/1119 and then update this repo.